tutorials/math/matrices_and_transforms.rst

```diff
@@ -606,14 +606,6 @@ this project which has colored lines and cubes to help visualize the
 :ref:`class_Basis` vectors and the origin in both 2D and 3D:
 https://github.com/godotengine/godot-demo-projects/tree/master/misc/matrix_transform
 
-.. note:: Spatial's "Matrix" section in Godot 3.2's inspector
-          displays the matrix as transposed, with the columns
-          horizontal and the rows vertical. This may be changed
-          to be less confusing in a future release of Godot.
-
-.. note:: You cannot edit Node2D's transform matrix directly in Godot 3.2's
-          inspector. This may be changed in a future release of Godot.
```

Comment:
The first note is no longer a problem in Godot 4.0, the 3D matrix on Node3D was corrected.

The second note is still relevant for Godot 4.0, the inspector does not let you edit the 2D matrix on Node2D.

Reply:
I see. So I think we can remove the first note, and keep the second note. Just have to update version number to 4.0.